server/handlers/get_account.go

```diff
@@ -28,6 +28,12 @@ func GetAccount(app *app.App) http.HandlerFunc {
 			panic(err)
 		}
 
+		oautAccounts, err := services.AccountOauthGetter(app.AccountStore, id)
```

Comment:
```suggestion
		oauthAccounts, err := services.AccountOauthGetter(app.AccountStore, id)
```

(but also, you might consider handling this in `services.AccountGetter`)

Reply:
I have moved the logic to `services.AccountGetter` 😄